portfolio/src/main/webapp/index.html

```diff
@@ -27,20 +27,29 @@
 			<div id="background-info-section">
 				<img src = "images/headshot.jpg" id = "headshot-img"/>
 				<p id="background-txt">I am originally from Clarksburg, Maryland but I am currently a
-					<span id = highlight>first generation</span> student
-					at the <span id = highlight>University of Michigan</span>
+					<span id = "highlight">first generation</span> student
+					at the <span id = "highlight">University of Michigan</span>
 					(<span id =  "go-blue">Go Blue!</span>). This summer I am working on a web application
 					as a <span id = "highlight">STEP intern</span> at Google.</p>
 			</div>
 			<div id="interests-section">
 				<p id="interests-txt">In my free time I enjoy <span id = "highlight">cooking</span> and seeking new
 					recipies online. I also enjoy
-					<span id = "highlight">strength-training</span> and <span id = highlight>yoga</span> as a way to
+					<span id = "highlight">strength-training</span> and <span id = "highlight">yoga</span> as a way to
 					relax. I have also been playing a lot of <span id = "highlight">low-stakes poker</span> with friends
 					during quarantine!</p>
 				<img src = "images/cooking.jpg" id = "cooking-img"/>
 			</div>
 		</div>
+		<div id = "img-slideshow-section">
+			<div id = "slideshow-header"><span id = "highlight">Where I Want To Be</span></div>
+			<div id = "slide">
```

Comment:
Up to you, but consider dynamically generating the img element, the div for the slide number, and the anchors.

In your DOMContentLoaded callback, you'll be able to then lookup id=slide and build things up from there. Bonus points if you encapsulate this into its own function that the DOMContentLoaded callback calls.

Reply:
Done. Should more refactoring be done or looks good?